Makefile

```diff
@@ -115,19 +115,21 @@ test-db-migrations:
 
 .PHONY: test-fast
 test-fast: node_version
-	BCRYPT=insecure npx mocha --recursive --fgrep @slow --invert
+	NODE_CONFIG_ENV=test BCRYPT=insecure npx mocha --recursive --fgrep @slow --invert
+
+NODE_CONFIG_ENV ?= test
```

Comment:
I think this defines the variable for the whole `Makefile`, so it might be more appropriate to define it at the top of the file.

Reply:
Alternatively I think you can define a variable for a specific target like this:

```Makefile
.PHONY: test-integration
test-integration: NODE_CONFIG_ENV ?= test
test-integration: node_version
	NODE_CONFIG_ENV=$(NODE_CONFIG_ENV) BCRYPT=insecure npx mocha --recursive test/integration
```